ui/src/components/icon/QIcon.js

```diff
@@ -167,6 +175,16 @@ export default Vue.extend({
       return h('svg', data, mergeSlot(this.type.nodes, this, 'default'))
     }
 
+    if (this.type.svguse === true) {
+      data.attrs.focusable = 'false' /* needed for IE11 */
+      data.attrs.viewBox = this.type.viewBox
+
+      const child = [
+        h('use', { attrs: { 'xlink:href' : this.type.src } }, mergeSlot(this.type.nodes,this, 'default'))
+      ]
+      return h('svg', data, child)
```

Comment:
I'm slightly torn on this one but I think I'd prefer to see it written like so to make things more readable (obviously replacing the `child` const.

```
return h('svg', data, [
  h('use', {
    attrs: {
      'xlink:href': this.type.src
    }
  }),
  mergeSlot(this.type.nodes,this, 'default')
])
```

Reply:
I totally agree with this. Not sure how I missed this one.